www-servers/automx2/automx2-2019.0.ebuild

```diff
@@ -21,6 +21,9 @@ BDEPEND="acct-user/automx2
 	)"
 RDEPEND="${BDEPEND}"
 
+DOCS="*.adoc doc/*.adoc contrib/*sample.conf"
```

Comment:
I see your comment regarding the docs being small enough to be installed unconditionally, but this breaks the usual expected behavior for users. Please put the docs behind the appropriate USE.

Reply:
My first ebuild version actually used the `doc` flag, and I removed it. Guess I'll revert that decision, won't I? 😉